src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -158,6 +158,9 @@ function useChatUtilsHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatUtilsHook {
       task: async () => {
         const saveChatPromises = response?.messages?.map((message) => {
           return new Promise((resolve) => {
+            // Skip if message_type is notification-deleted
+            if (message?.message_type === 'notification-deleted') return resolve(true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for skipping the message with type 'notification-deleted' is correct. However, it's important to ensure that this doesn't introduce any side effects in other parts of the application that might be expecting this message type to be present in the local database.

```diff
+    // Skip if message_type is notification-deleted
+    if (message?.message_type === 'notification-deleted') return resolve(true);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->